src/controllers/idpMetadataController.ts

```diff
@@ -0,0 +1,21 @@
+import { Task } from "fp-ts/lib/Task";
+import {
+  IResponseSuccessJson,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { SuccessResponse } from "../types/commons";
+import { log } from "../utils/logger";
+
+export default class IdpMetadataController {
+  constructor(private readonly idpMetadataRefresher: () => Task<void>) {}
+
+  public async refresh(): Promise<IResponseSuccessJson<SuccessResponse>> {
+    // tslint:disable-next-line: no-floating-promises
```

Comment:
Probably if we add a catch() we don't need to disable linting

Reply:
Fixed in 8019d63